src/index.ts

```diff
@@ -1,8 +1,138 @@
-import { Bot } from './app/bot';
 import dotenv from 'dotenv';
+import { IContainer } from './common/types';
+import { readdir } from 'fs/promises';
+import { Listener } from './app/listener';
+import express from 'express';
+import { Plugin } from './common/plugin';
+import path from 'path';
+import { ISlashCommand, SlashCommand, slashCommands } from './common/slash';
+import Bottle from 'bottlejs';
+import { Container } from './bootstrap/container';
 
 // Load env vars in.
 dotenv.config();
 
-const app = new Bot();
-app.run();
+(async () => {
+  const containerBuilder = new Bottle();
+  new Container(containerBuilder);
+  containerBuilder.resolve({});
+  const container = containerBuilder.container as IContainer;
+  const listener = new Listener(container);
+  const webServer = express();
+  try {
+    container.loggerService.info('Loading and running Bot...');
+
+    container.clientService.on('ready', async () => {
+      container.pluginService.reset();
+
+      // load classic plugins
+      const pluginFolder = path.join(__dirname, '/plugins');
+      const pluginFiles = await readdir(pluginFolder);
```

Comment:
My checkout is crashing on this line
```
error: Could not read plugins folder: Error: ENOENT: no such file or directory, scandir 'C:\Users\Tanner\Desktop\Projects\Github\lion\src\plugins'
```
The `__dirname` mustve changed since we are in a different file now. The path should be `lion\src\app\plugins`

Reply:
Oh good catch, I didn't notice this when remerging it

Reply:
I wish there was a "run a command" e2e test that would catch stuff like this. All the existing tests passed despite this